bn-apps/settings.gradle

```diff
@@ -1,2 +1,2 @@
-include ':memberships-management:membership-service'
-include ':memberships-management:membership-service-contracts-and-states'
```

Comment:
`:memberships-management:membership-service-contracts-and-states` should stay otherwise the project wouldn't build